src/connectors/faiss_file.py

```
@@ -46,58 +203,115 @@ def __init__(self, metadata, service):
         self.service = service
 
     def get_text(self) -> str:
-        file_id = self.metadata['id']
-        path_lower = self.metadata['path_lower']
-        mime_type = self.metadata['mimeType']
+        file_id = self.metadata["id"]
+        path_lower = self.metadata["path_lower"]
+        mime_type = self.metadata["mimeType"]
 
         try:
             meta, resp = self.service.files_download(file_id or path_lower)
```

Ruff (0.14.10)

Unpacked variable `meta` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)